protocol/messenger_communities.go

```diff
@@ -1552,7 +1552,7 @@ func (m *Messenger) acceptRequestToJoinCommunity(requestToJoin *communities.Requ
 			return nil, err
 		}
 
-		grant, err := community.BuildGrant(pk, "")
+		grant, err := community.BuildGrant(pk)
```

Comment:
@osmaczko @jrainville just to clarify: is there still a need for a grant for `acceptRequestToJoinCommunity `

Reply:
I think yes, you'll need that grant for the profile showcase. Attaching it to `acceptRequestToJoinCommunity` will ensure you receive a grant with the acceptance, mitigating any timing issues.